crates/re_viewer_context/src/component_ui_registry.rs

```diff
@@ -633,3 +644,11 @@ fn try_deserialize<C: re_types::Component>(value: &dyn arrow2::array::Array) ->
         }
     }
 }
+
+fn none_or_many_values_ui(ui: &mut egui::Ui, num_instances: usize) {
+    if num_instances == 1 {
+        ui.label("(empty)");
```

Comment:
this deserves a comment too, as this looks very wrong to my eyes

Reply:
oops...